redisgears_core/src/lib.rs

```diff
@@ -868,17 +933,16 @@ fn initialize_v8_backend(
 }
 
 pub(crate) fn get_backend(
-    ctx: &Context,
     name: &str,
 ) -> Result<&'static mut Box<dyn BackendCtxInterfaceInitialised>, RedisError> {
     get_backends_mut()
         .get_mut(name)
-        .ok_or(RedisError::Str("No such backend"))
+        .ok_or(RedisError::String(format!("No such backend: {name}")))
         .or_else(|_| {
             let uninitialised_backend = get_uninitialised_backends_mut()
                 .remove(name)
-                .ok_or_else(|| RedisError::String(format!("Unknown backend {}", name)))?;
-            let backend = initialize_v8_backend(ctx, uninitialised_backend)?;
+                .ok_or_else(|| RedisError::String(format!("Unknown backend {name}")))?;
+            let backend = initialize_v8_backend(uninitialised_backend)?;
```

Comment:
I think this should still get a ctx to indicate that you can only do it when the Redis GIL is held. At least until we will wrap the globals with RedisGILProtected object.